docs/general/performance.md

```diff
@@ -28,6 +28,10 @@ new Chart(ctx, {
 });
 ```
 
+## Provide ordered data
```

Comment:
I would probably combine this with the "Data structure and format" section since it only takes effect when `parsing: false`

Reply:
Nevermind. It looks like that's no longer true